core/src/commonMain/kotlin/com/xebia/functional/xef/llm/models/embeddings/EmbeddingModel.kt

```diff
@@ -0,0 +1,5 @@
+package com.xebia.functional.xef.llm.models.embeddings
+
+enum class EmbeddingModel(val modelName: String) {
+  TextEmbeddingAda002("text-embedding-ada-002")
```

Comment:
```suggestion
  TEXT_EMBEDDING_ADA_002("text-embedding-ada-002")
```

Idem, this should use streaming snake case according to the official recommendations.

Reply:
Addressed in https://github.com/xebia-functional/xef/pull/196/commits/c5a70f0ecea5e0df60ce517109115a4b9a8ce956